src/Test/IntegrationTestCase.php

```diff
@@ -211,7 +217,7 @@ protected function doIntegrationTest($file, $message, $condition, $templates, $e
                     return;
                 }
 
-                throw new Error(sprintf('%s: %s', \get_class($e), $e->getMessage()), -1, null, $e);
+                throw new Error(sprintf('"%s": "%s".', \get_class($e), $e->getMessage()), -1, null, $e);
```

Comment:
this change looks wrong to me.

Reply:
fabbot.io still wants this to be changed. How do I deal with this?

Reply:
you ignore it. this exception message formatting check has some false positives when we have special needs